server/src/test/java/org/opensearch/cluster/metadata/MetadataCreateIndexServiceTests.java

```diff
@@ -90,18 +90,7 @@
 import org.opensearch.threadpool.ThreadPool;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
No wildcard imports please

Reply:
need to remove IDE auto corrects. Thanks for pointing out